sdk/keyvault/azkeys/models.go

```diff
@@ -94,10 +76,10 @@ func keyAttributesFromGenerated(i *generated.KeyAttributes) *KeyAttributes {
 	}
 }
 
-// KeyBundle - A KeyBundle consisting of a WebKey plus its attributes.
-type KeyBundle struct {
-	// The key management attributes.
-	Attributes *KeyAttributes `json:"attributes,omitempty"`
+// KeyVaultKey - A KeyVaultKey consists of a WebKey plus its attributes.
+type KeyVaultKey struct {
```

Comment:
The motivation for this name in other SDKs was to prevent confusion with `Key` types from other namespaces. In a Go application the type will be qualified, so I think there's a case for `azkeys.Key` here.

Reply:
@heaths thoughts?

Reply:
@chlowell is right, but I don't see a strong reason to deviate from naming without due cause (e.g. some name happens to be a keyword in a language), but perhaps @JeffreyRichter has a strong opinion on this either way.

Reply:
I don't love this name. Could it be VaultKey in the azkeys package?

Reply:
I think `Key` is better than `VaultKey`